installers-conda/build_installers.py

```diff
@@ -226,18 +227,16 @@ def _definitions():
         ],
         "conda_default_channels": ["conda-forge"],
         "specs": [
-            "python",
-            "conda",
+            f"python={PY_VER}",
             "mamba",
-            "pip",
         ],
         "installer_filename": OUTPUT_FILE.name,
         "initialize_by_default": False,
         "initialize_conda": False,
         "register_python": False,
         "license_file": str(RESOURCES / "bundle_license.rtf"),
         "extra_envs": {
-            f"spyder-{SPYVER}": {
+            "spyder-rt": {
```

Comment:
A couple of questions about this change:

* This environment won't appear in the list of envs listed in `Preferences > Main interpreter`, right?
* What if we name it `spyder-runtime` instead to a bit more explicit about it?

Reply:
Correct, `spyder-6` (base) and `spyder-6/envs/spyder-rt` (spyder-rt) will not show up in the custom interpreter list.
No problem, I'll rename to `spyder-runtime`.